src/pages/design/[sectionId]/[articleId].tsx

```diff
@@ -27,17 +31,96 @@ export const getStaticProps: GetStaticProps = async (context) => {
 };
 
 export const ArticlePage = ({sectionId, articleId}: {sectionId: string; articleId: string}) => {
-    const section = sections.find((item) => item.id === sectionId);
+    const router = useRouter();
+    const section = designSections.find((item) => item.id === sectionId);
     const article = section?.articles.find((item) => item.id === articleId);
 
     if (!section || !article) {
         return null;
     }
 
+    const sections = useMemo<Section[]>(() => {
+        const result: Section[] = designSections.map((section) => ({
+            id: section.id,
+            title: section.title,
+            // Uncomment it to show overview tab
+            // url: `/design/${section.id}`,
+            subSections: section.articles.map((article) => ({
+                id: article.id,
+                title: article.title,
+                url: `/design/${section.id}/${article.id}`,
+            })),
+        }));
+        return result;
+    }, []);
+
+    const currentSection = useMemo(
```

Comment:
Why do we calculate `prevSection` and `nextSection` here at the page level? The same logic for components is placed at the page component level.

Let's move these `useMemo`'s into the `DesignArticle` component.